src/hacbs/components/Commits/CommitsGettingStartedModal.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import { Button, Modal, ModalVariant, Text, Title } from '@patternfly/react-core';
+import imageUrl from '../../imgs/app-workflow.svg';
+
+type CommitsGettingStartedModalProps = {
+  shown: boolean;
+  onHide: () => void;
+};
+
+const CommitsGettingStartedModal: React.FC<CommitsGettingStartedModalProps> = ({
+  shown,
+  onHide,
+}) => {
+  const header = (
+    <Title className="pf-u-text-align-center" headingLevel="h2">
+      Getting started with Commit view
```

Comment:
```suggestion
      Getting started with commits view
```

Reply:
Maybe `Getting started with the Commit view`? The view is for a single commit.

Reply:
@Ranelim @MariaLeonova please advise.

But really it's not that big of a deal as we'll be sweeping text later when microcopy is updated.